uli-community/lib/uli_community_web/live/dashboard_live.ex

```diff
@@ -7,15 +7,37 @@ defmodule UliCommunityWeb.DashboardLive do
   def mount(_params, _session, socket) do
     if connected?(socket), do: Phoenix.PubSub.subscribe(UliCommunity.PubSub, @topic)
 
-    slurs = UserContribution.get_top_slurs(10)
-    severity_data = UserContribution.get_severity_distribution()
+    slurs =
+      try do
+        UserContribution.get_top_slurs(10)
+      rescue
+        _ -> nil
+      end
+
+    severity_data =
+      try do
+        UserContribution.get_severity_distribution()
+      rescue
+        _ -> nil
+      end
 
     {:ok, assign(socket, slurs: slurs, severity_data: severity_data)}
   end
 
   def handle_info(:slur_changed, socket) do
-    slurs = UserContribution.get_top_slurs(10)
-    severity_data = UserContribution.get_severity_distribution()
+    slurs =
```

Comment:
@aatmanvaidya 
We don't have to build realtime components for every feature. This page does not need to react to any changes.